docker/install_ruby.sh

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash
+
+echo "Installing RVM/ruby dependencies..."
+apt-get install -y openssh-server git-core openssh-client build-essential openssl libreadline7 libreadline-dev curl zlib1g zlib1g-dev libssl-dev libyaml-dev libsqlite3-dev sqlite3 libxml2-dev libxslt1-dev autoconf libc6-dev libncurses5-dev automake libtool bison subversion pkg-config
```

Comment:
Would it be too much to just include this stuff in the Dockerfile? I know these are only used for this one script, but once the image is built we wouldn't have to wait any extra time for all of this to install. 

Of course it would add to the time it takes to actually build the image. Food for thought

Reply:
that's a fantastic idea 👌

Reply:
how would you feel about pulling the whole thing over to straight-up debian or even alpine in that case?  pull the maven stuff in by hand just like the ruby stuff.  again would add slightly to the build time, but makes more "sense" in a way, and if we use an alpine box it might just even out

Reply:
I'm alright with that as well. What made me not do that initially was the Maven image was `curl`ing the binary down and I just though "leave well enough alone"